### PR TITLE
Garrote sprite fix

### PR DESCRIPTION
**Pull request by Spacedong**
Fixes garrotes being invisible, reworks the color system to be in line with the way cable and cablecuffs work
:cl: Spacedong
fix: Garrotes are no longer invisible.
/:cl:

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
Did you test this and see that it works?

File: hippiestation/code/game/objects/items/garrote.dm [hunk 1]
```diff
@@ -32,22 +32,57 @@
...
 	var/garroting = FALSE
 	var/next_garrote = 0
 
-/obj/item/garrote/Initialize()
-	..()
+/obj/item/garrote/Initialize(mapload, param_color)
+	.=..()
+
+	var/list/cable_colors = GLOB.cable_colors
+	item_color = param_color || item_color || pick(cable_colors)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/garrote.dm:44]: > cable_colors

what?

Comment (Spacedong) [reply hippiestation/code/game/objects/items/garrote.dm]: it works

**Comment by Spacedong** [comment]
@GuyonBroadway yes

File: hippiestation/code/game/objects/items/garrote.dm (hunk 1 above)
Comment (Spacedong) [reply hippiestation/code/game/objects/items/garrote.dm]: GLOBAL_LIST_INIT(cable_colors, list(
	"yellow" = "#ffff00",
	"green" = "#00aa00",
	"blue" = "#1919c8",
	"pink" = "#ff3cc8",
	"orange" = "#ff8000",
	"cyan" = "#00ffff",
	"white" = "#ffffff",
	"red" = "#ff0000"
))

**Comment by Pyko1** [comment]
are the catbeasts random cable color now?

**Pull request merged**